### PR TITLE
fixed display of active cases

### PR DESCRIPTION
**Pull request by nerdmaennchen**
somehow the display of worldwide total active cases was broken.
This PR fixes the issue

chears

### CODE REVIEW

File: MMM-COVID19.js [hunk 1]
```diff
@@ -76,6 +76,10 @@ Module.register("MMM-COVID19", {
     return this.config.header
   },
 
+  numberWithCommas: function(x) {
```
Comment (bibaldo) [inline MMM-COVID19.js]: this should be places at the bottom next to `compareValues` and please put some comments before the function

File: MMM-COVID19.js [hunk 2]
```diff
@@ -137,7 +141,7 @@ Module.register("MMM-COVID19", {
           deaths = globalStats["total_deaths"],
           newDeaths = globalStats["new_deaths"],
           totalRecovered = globalStats["total_recovered"],
-          activeCases = '';
+          activeCases = (cases && totalRecovered)?this.numberWithCommas(parseInt(cases.replace(",","")) - parseInt(totalRecovered.replace(",",""))):"";
```
Comment (bibaldo) [inline MMM-COVID19.js]: Could you please indent it properly? So we don't have long single lines
I would suggest you:

```
activeCases = (cases && totalRecovered) ? 
  this.numberWithCommas(parseInt(cases.replace(",","")) - parseInt(totalRecovered.replace(",","")))
  : "";

**Comment by bibaldo** [comment]
> somehow the display of worldwide total active cases was broken.
> This PR fixes the issue
> 
> chears

nono, it was only a design decision. I will approve your PR if you perform the suggested modifications

File: MMM-COVID19.js (hunk 1 above)
Comment (nerdmaennchen) [reply MMM-COVID19.js]: done in 803276d

File: MMM-COVID19.js (hunk 2 above)
Comment (nerdmaennchen) [reply MMM-COVID19.js]: done in 803276d

**Pull request merged**